clang-tools-extra/test/clang-tidy/checkers/modernize/raw-string-literal.cpp

```diff
@@ -105,7 +105,8 @@ char const *const StringizedMacroArgument = HAT(foo\\bar);
 
 #define SUBST(lit_) lit_
```

Comment:
what about concat string in macro?
e.g.
```
#define SUBST(a,b) a#b
char const *const MacroArgument = SUBST("foo\\bar","cd");
```

Reply:
Good point, this doesn't work. It wants to create a replacement `"R\"(foo\\bar\\baz\\bazz\\foo\"cd\")\""` which is non sense.

Most likely macros should be handled in a `PPCallback` instead, thus it's better to drop this PR and implement this feature there instead.